elk/training/burns_norm.py

```diff
@@ -0,0 +1,47 @@
+import torch
+from torch import Tensor, nn
+
+
+class BurnsNorm(nn.Module):
+    """Burns et al. style normalization. Minimal changes from the original code."""
+
+    def __init__(self, scale: bool = True):
+        super().__init__()
+        self.scale: bool = scale
+
+    def forward(self, x: Tensor) -> Tensor:
+        """Normalizes per template
+        Args:
+            x: input of dimension (n, v, c, d) or (n, v, d)
+        Returns:
+            x_normalized: normalized output
+        """
+        num_elements = x.shape[0]
+        x_normalized: Tensor = x - x.mean(dim=0) if num_elements > 1 else x
+
+        if not self.scale:
+            return x_normalized
+        else:
+            std = torch.linalg.norm(x_normalized, dim=0) / torch.sqrt(
+                torch.tensor(x_normalized.shape[0], dtype=torch.float32)
```

Comment:
Why do we need to wrap this in a Tensor here? You could do 
```py
torch.linalg.norm(x_normalized, dim=0) / x_normalized.shape[0] ** 0.5
```
This is also going to fail if `x` is not `float32`. Probably not a huge deal but I'd mildly prefer for the code to be agnostic to that

Reply:
done, thanks